### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by ernichechelski**
Explicit warning to not use dashes in the keys.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,6 +34,8 @@ plugin 'cocoapods-keys', {
     ...
   ]}
 ```
+> Please do not use dash in key names ([Reason why here in this issue #197](https://github.com/orta/cocoapods-keys/issues/197)).
+<br>For example convert any key like this `WRONGLY-DEFINED-KEY` to `CorrectlyDefinedKey`.
```
Comment (ashfurrow) [inline README.md:38]: We can do this without inline HTML, I think:

```suggestion
> Please do not use dash in key names ([Reason why here in this issue #197](https://github.com/orta/cocoapods-keys/issues/197)).
> 
> For example convert any key like this `WRONGLY-DEFINED-KEY` to `CorrectlyDefinedKey`.
```

**Review by ashfurrow — APPROVED** [review]
Looks good, just one small note!

**Comment by ashfurrow** [comment]
Thank you!

**Comment by ernichechelski** [comment]
> Thank you!

Thanks for accepting this change 😄☺️

**Pull request merged**